protostar/cheatable_starknet/controllers/expect_call_controller.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from protostar.testing.test_environment_exceptions import ExpectedCallException
+from protostar.starknet import CairoOrPythonData, Address
+from protostar.starknet.selector import Selector
+
+if TYPE_CHECKING:
+    from protostar.testing import Hook
+    from protostar.cheatable_starknet.cheatables.cheatable_cached_state import (
+        CheatableCachedState,
+    )
+
+
+@dataclass
+class ExpectedCall:
+    address: Address
+    fn_selector: Selector
+    calldata: CairoOrPythonData
+
+
+def remove_expected_call_external(
```

Comment:
what does the name mean?

Reply:
[change remove expected call from external to static](https://github.com/software-mansion/protostar/pull/1481/commits/1179d77e4e82e4d969ac27da7712c09d39001975)